packages/Molecules/Form/stories.js

```diff
@@ -11,7 +11,7 @@ storiesOf('Molecules/Form', module)
   .addDecorator(VueInfoAddon)
   .add('Form', withReadme(readme, () => ({
     components: { RplForm },
-    template: `<rpl-form :formData="formData"></rpl-form>`,
+    template: `<rpl-form :formData="formData" :submitHandler="submitHandler" :is-new-model="isNewModel" :options="options"></rpl-form>`,
```

Comment:
Should there be some consistency between prop naming, should `:is-new-model` be `:isNewModel`?